README.md

```diff
@@ -52,6 +52,8 @@ project.schema     #the data schema for the project
 project.templates  #a collection of templates for this project
 
 template = project.templates.find('template-slug')                             #fetch a template by slug
+template.test({a_data_point: 1, another_one: 'Tuesday'})                       #test your implementation
+ > nil # if no errors found, otherwise return RuntimeError
```

Comment:
I feel like this explanation doesn't say a whole lot...It took me a few seconds to realize the purpose of this endpoint (don't incur a billing event I take it), so a more descriptive explanation would help.

Maybe have two sections under `Usage`:  one being `Testing` and the other for `Generating Content`, and each one having a sentence or two (rather than code comments). Just a suggestion though...we can talk about this in-person.